timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -390,15 +409,15 @@ def predict(
         model: Optional[str] = None,
         **kwargs,
     ) -> TimeSeriesDataFrame:
-        """Return quantile and mean forecasts given a dataset to predict with.
+        """Return quantile and mean forecasts for the given dataset, starting from the end of each time series.
 
         Parameters
         ----------
         data : TimeSeriesDataFrame
             Time series data to forecast with.
-        model : str, default=None
-            Name of the model that you would like to use for forecasting. If None, it will by default use the
-            best model from trainer.
+        model : str or None, default = None
```

Comment:
If we're being nitpicky with this, the style guide says to use:
```
model : str, optional
```
https://numpydoc.readthedocs.io/en/latest/format.html#parameters